### DISCUSSION TITLE
How to order keymap in help?

### DISCUSSION THREAD

**Discussion by fjchen7** [nvim-tree/nvim-tree.lua#1745] [Q&A] [answered]
keymaps show by alphabetic order in the action name.
![image](https://user-images.githubusercontent.com/10106636/201815941-be3c2329-f807-4192-9327-3a728380d83a.png)

It is a bit hard to take a quick galance. How to reorder them? like order by key. I would like to let lowercase letter in first, and then uppercase, and finally `<C-*>` key

**Comment by alex-courtis** [comment, marked as answer]
It's coming #1579

![20221115_145855 429x1541](https://user-images.githubusercontent.com/1505378/201823119-10fad895-aaf7-46de-aa1a-3d0f3ccc1c77.png)
